### DISCUSSION TITLE
Accessing node color values?

### DISCUSSION THREAD

**Discussion by jimmyff** [zathras/jovial_svg#111] [General]
I'm finding it quite tricky to access the color values from SVG nodes via their paint field, I'm currently parsing their toString output! I only need to grab these occasionally for editing purposes so it's no big deal but I'm surely missing an easier way?

**Comment by zathras** [comment]
See https://github.com/zathras/jovial_svg/issues/115

**Reply by jimmyff** [reply]
Great thanks, I'll track that issue
